src/views/main/MainView.tsx

```diff
@@ -6,6 +6,7 @@ import { withLazy } from 'shared/utils';
 import { Navbar, Sidebar } from '.';
 
 import csx from './MainView.scss';
+import TemplateDetailsView from 'viewstemplates/template-details';
```

Comment:
Use withLazy HOC for dynamically loading this code. Also change path to views/template-details.